packages/manager/apps/pci-block-storage/src/pages/detach/DetachStorage.page.tsx

```diff
@@ -33,21 +163,8 @@ export default function DetachStorage() {
     isPending: isInstancesPending,
   } = useAttachedInstances(projectId, volumeId);
 
-  const [selectedInstanceId, setSelectedInstanceId] = useState<
-    TAttachedInstance['id'] | null
-  >(null);
-
-  const selectedInstance = useMemo(() => {
-    if (!!selectedInstanceId && !!instances)
-      return instances.find(({ id }) => id === selectedInstanceId)!;
-    return null;
-  }, [selectedInstanceId, instances]);
-
   const { detachVolume, isPending: isDetachPending } = useDetachVolume({
-    projectId,
-    volumeId,
-    instanceId: selectedInstanceId,
-    onError(err: Error) {
+    onError(err) {
```

Comment:
If `error` is an instance of AxiosError, you could not access `error?.message` so it could be useful to test it at hook level and return either the good shape or only the message

Reply:
There is an error in the typing, error cannot be nullish, so there is always an error with a message

We should see later for improvement on this I think